### ISSUE TITLE
sftlf

### ISSUE THREAD

**Issue by oligan** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#265]
I would just like confirmation that sftlf (land area fraction) also includes the fraction of land ice. 

1 | landf | land_area_fraction | fraction of the grid  cell occupied by land | 1.0 | Please express   "X_area_fraction" as the fraction of horizontal area occupied by X. | None |   | None | real | longitude latitude | sftlf | atmos | fx | area: areacella | CMIP5 [fx] ((isd.002)) | DCPP,PMIP | CMIP6,DCPP,DECK,PMIP
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by taylor13** [comment]
Jonathan Gregory, @martinjuckes and I had a very long series of the email exchanges discussing this in December 2016.  I know that land ice resting on a solid surface that is above sea level is considered "land".  I can't remember what we decided about ice-shelves, either floating or grounded.  Will try to review the outcome and respond further.

**Comment by martinjuckes** [comment]
I believe we concluded that there is some room for flexibility in the CF Convention, so that slightly different interpretations may hold in different contexts. Karl was clear that `sea` extent should vary with depth, and that `land` and `sea` on any given surface should be complementary. It would follow that `land` can have different extents below and above ice sheets. 
When looking at atmospheric diagnostics above a 1km thick ice sheet, neither the grounding line nor the line defined by a basal height of sea level is of any relevance: we need diagnostics on a spatial area which covers the whole of the ice sheets. I think we decided that `land` could stretch to this interpretation.
